### ISSUE TITLE
Instructions for integration on an external template

### ISSUE THREAD

**Issue by exaline-ru** [leeliwei930/godspeed-essentials#2]
Instructions for integration on an external template (not Flametree Theme).

Your plugin is very interesting, but when installed on my template, no data is output. 

Some borrowing from your template doesn't help.

Thanks for attention!

**Comment by leeliwei930** [comment]
Yeah, since this our FYP project, and our team have limited resources working on it, as you mention the issue above due to we are using mostly VueJS components to render the frontend view as you can see the October plugin component is using full of custom DOM element, but anyways thanks for your feedback.

**Comment by leeliwei930** [comment]
But for my recommendation, you can fork our plugin component markup and rewrite some of the HTML to make it works.
